frontend/src/component/admin/users/InviteLinkBar/InviteLinkBar.tsx

```diff
@@ -15,24 +14,5 @@ export const InviteLinkBar: VFC = () => {
         });
     };
 
-    return (
```

Comment:
This bordering and style will be used in 2 places, so moving it to shared component on bottom. Also all of these stylings were not needed.